kickstarter/android-oss
# NT-1951: UI/UX – Error loading initial comments

# 📲 What

UI/UX – Error loading initial comments

# 🤔 Why

- Because every now and then things will fail, we need to handle problems loading comments.

# 🛠 How

- Created custom error layout item
- Added custom item to viewholder
- Added Initial Load Error observable
- Subscribed to initialCommentLoadError
- Added unit tests for initial comment load error

# 👀 See

https://user-images.githubusercontent.com/63934292/121552959-c8a90000-ca08-11eb-93b9-8a6311506cb4.mp4

| Before 🐛 | After 🦋 |
| --- | --- |
|  |  |

# 📋 QA

- Select any project with comments
- Turn off network connect
- Select comments, you should see an error page on failure

# Story 📖

https://kickstarter.atlassian.net/browse/NT-1951
